### ISSUE TITLE
Assertion failed on parsing malformed CSS file (main/entry.c:1108)

### ISSUE THREAD

**Issue by kcwu** [universal-ctags/ctags#530]
How to reproduce
```
$ echo -e '0["\t' > min.css
$ ctags min.css
ctags: main/entry.c:1108: void initTagEntryFull(tagEntryInfo *const, const char *const, unsigned long, const char *, fpos_t, const char *): Assertion `name == NULL || strchr (name, '\t') == NULL' failed.
ctags: main/entry.c:1108: parsing min.css:1 as CSS
Aborted
```

this is found by afl-fuzz

**Comment by b4n** [comment]
Indeed.  What are parsers supposed to do here, replace `\t` with a space?

**Comment by b4n** [comment]
Knowing the `\t` is technically legitimate for a well formed tag like `foo[a^="	"] {}`

**Comment by masatake** [comment]
How about applying the rule of tagfied of format.rst to the name field:
```
* The value may be empty.
  It cannot contain a <Tab>.

  - When a value contains a "\\t", this stands for a <Tab>.
  - When a value contains a "\\r", this stands for a <CR>.
  - When a value contains a "\\n", this stands for a <NL>.
  - When a value contains a "\\\\", this stands for a single '\\' character.
```

About the other control character, replacing it with a space you proposed in 8b3b567 may be the way we can choose.

Another question is which layer should handle this characters. My ideas are: each parsers are expected to handle the characters, and main handles them with warning.

**Comment by masatake** [comment]
Solved by #582. Thank you.
